File: launchdarkly/python/src/GetHoldoutByIdExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.HoldoutsBetaApi(api_client).get_holdout_by_id(
            project_key=None,
            environment_key=None,
            holdout_id=None,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling HoldoutsBeta#get_holdout_by_id: %s\n" % e)

File: launchdarkly/ruby/src/GetHoldoutByIdExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::HoldoutsBetaApi.new.get_holdout_by_id(
        "projectKey_string", # project_key
        "environmentKey_string", # environment_key
        "holdoutId_string", # holdout_id
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling HoldoutsBetaApi#get_holdout_by_id: #{e}"
end
